src/combineReducers.js

```diff
@@ -106,8 +106,11 @@ export default function combineReducers(reducers) {
     const key = reducerKeys[i]
 
     if (process.env.NODE_ENV !== 'production') {
-      if (typeof reducers[key] === 'undefined') {
+      const reducerUnderInspection = reducers[key]
```

Comment:
You shouldn't need to extract this out. Just keep using `reducers[key]`. It's probably not as fast, but this is dev-only code.